### PR TITLE
Add autoscrolling and improve page formatting

### PR DESCRIPTION
**Pull request by BRHoward**

### CODE REVIEW

**Comment by BRHoward** [comment]
@jebbinSCL

File: public/css/main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.conversation-window {
+	height: 600px
+}
```
Comment (jebbinSCL) [inline public/css/main.css]: Looks like you forgot the new lines of most of these files!

**Comment by jebbinSCL** [comment]
Only issue is new lines at the end of files

**Comment by jebbinSCL** [comment]
:ship: after tests complete

**Pull request merged**